src/QMCWaveFunctions/Fermion/MatrixDelayedUpdateCUDA.h

```diff
@@ -208,7 +196,7 @@ class MatrixDelayedUpdateCUDA
     auto& cone_vec                   = mw_mem.cone_vec;
     auto& czero_vec                  = mw_mem.czero_vec;
     auto& prepare_inv_row_buffer_H2D = mw_mem.prepare_inv_row_buffer_H2D;
-    const int norb                   = engine_leader.get_psiMinv().rows();
+    const int norb                   = engine_leader.invRow.size();
```

Comment:
If feel like getting this value from the passed psiMInv_ref is less of a change and depends on more straight forward input. But It seems like consistency between psiMInv rows and invRow(s) also must be maintained.

Reply:
It is a stopgap for the moment. I will actual be able to store norb as a const via the constructor since DiracDeterminantBatched also have the size ironed at the constructor. This was never possible when MatrixDelayedUpdateCUDA was first introduced.